src/include/storage/data_table.h

```diff
@@ -122,7 +148,8 @@ class DataTable {
    * @param layout the initial layout of this DataTable. First 2 columns must be 8 bytes.
    * @param layout_version the layout version of this DataTable
    */
-  DataTable(common::ManagedPointer<BlockStore> store, const BlockLayout &layout, layout_version_t layout_version);
+  DataTable(common::ManagedPointer<BlockStore> store, const BlockLayout &layout,  // NOLINT
```

Comment:
Why were these needed?

Reply:
I didn't catch this one (side effect of a change made and reverted elsewhere).  This should be reverted as well.

Reply:
👍

Reply:
done

Reply:
Apparently gcc doesnt like it: http://jenkins.db.cs.cmu.edu:8080/blue/organizations/jenkins/terrier/detail/PR-850/116/pipeline#step-231-log-214

Reply:
so i reverted it back